tests/unit/ui/CheckMkAsyncSelect.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { SelectableValue } from '@grafana/data';
+import { render } from '@testing-library/react';
+import * as React from 'react';
+
+import { CheckMkAsyncSelect } from '../../../src/ui/components';
+
+describe('CheckMkAsyncSelect', () => {
+  const autocompleter = jest.fn(async (prefix: string): Promise<Array<SelectableValue<string>>> => {
+    if (prefix === 'sentinel') {
+      return [{ label: 'sentinel', value: 'sentinel' }];
+    }
+    return [];
+  });
+
+  it("uses the specific options if the default ones don't include the value", async () => {
+    const screen = await render(
+      <CheckMkAsyncSelect autocompleter={autocompleter} inputId={'foo'} onChange={() => undefined} value="sentinel" />
+    );
+
+    expect(autocompleter).toHaveBeenCalledWith('');
+    expect(autocompleter).toHaveBeenCalledWith('sentinel');
```

Comment:
i removed a assertion here, as it was causing some trouble.

```
    expect(screen.findByText('sentinel')).toBeInTheDocument();
```

the error also happened in the ci:

https://github.com/tribe29/grafana-checkmk-datasource/actions/runs/4291296286/jobs/7476305089